recipes/jwt-cpp/all/patches/0001-fix-windows-warning.patch

```diff
@@ -0,0 +1,69 @@
+diff --git a/include/jwt-cpp/base.h b/include/jwt-cpp/base.h
```

Comment:
Remove this patch and apply that one related to OpenSSL. Warnings are not errors and it's part of the upstream. If something is wrong, report to the upstream first.

Reply:
Alrighty, yeah I got a PR trying to fix them.